factorCombinations.py

```diff
@@ -0,0 +1,24 @@
+class Solution(object):
+    import math
+    def getFactors(self, n):
+        """
+        :type n: int
+        :rtype: List[List[int]]
+        """
+        if n <= 3:
+            return []
+        res = []
+        m = int(math.sqrt(n))
+        for i in range(2, m + 1):
+            if n % i == 0:
+                num1, num2 = i, n // i
+                res.append([num1, num2])
+                res2 = self.getFactors(num2)
```

Comment:
Maybe should make a dictionary to avoid duplicated calculations.
e.g., if n is 64, you might have to get the factor of 16 or 8 quite a few times